releasenotes/notes/fix-broken-a5-import-a2358410cf4a37da.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Agent 5 configuration import was missing critical parameters.
```

Comment:
what was the user impact? Would be nice to explain this in more details if possible

Reply:
Having `config.Set` private would indeed be a valuable move. I will keep the wrapping logic in that extend. I updated the PR description with a more detailed explanation.